controllers/auth.controller.js

```diff
@@ -1,8 +1,13 @@
 const User = require("../models/user.model");
-
+const { v4: uuidV4 } = require("uuid");
```

Comment:
Mozesz zwrocic uwage na org importow
https://kamilmrowka.com/posts/import-organization

Reply:
Podoba mi się jak piszesz commits a potem robisz to co napsiałeś. Dobrze się czyta taki kod. Tak trzymaj.

Reply:
dzięki za namiary na artykuł, uporządkuję importy według Twoich wskazówek, różna długość ids wynika z tego, że korzystałem początkowo z innej metody